browser/net/BUILD.gn

```diff
@@ -22,9 +22,6 @@ buildflag_header("geolocation") {
 }
 
 source_set("net") {
```

Comment:
This explicit target makes no sense, because it uses things from `//chrome/browser` and `//brave/browser` a lot. Because of that, it should be a source list `//brave/browser/net/sources.gni` that must be included into `//brave/browser/sources.gni`.

`check_includes = false` and `// nogncheck` are basically the same thing, even if it only covers few files. `// nogncheck` may be used in some rare cases like adding an include that's not properly added on some platform or implicitly appears because of invalid GN dependency that's out of our control. Here we have a clear issue that should be solved with `source.gni` approach.

Reply:
> This explicit target makes no sense, 

Yeah, perfectly makes sense. I'm going to remove `:net` target here and leave only test targets.